File: VesNing.BabyWebsite/Domain/Model/JournalModel.cs
using Infrastructure.Identity;
using Microsoft.AspNet.Identity.EntityFramework;
using System;
using System.Collections.Generic;
using System.Text;

namespace Domain.Model
{
   public class JournalModel
    {
       
        public string JournalID
        {
            get; set;
        }
        public string Title
        {
            get; set;
        }
        public string Content
        {
            get; set;
        }
        public DateTime JournalDate
        {
            get;set;
        }
        public string UserId
        {
            get;set;
        }
        public AppUser User
        {
            get; set;
        }
    }
}

File: VesNing.BabyWebsite/BabyWebsite/src/app/baby-journal/journal.model.ts
export class JournalModel
{
    constructor()
    {
    }
    //日志标识
    journalID:string;
    //日志链接，后期考虑使用路由
    href:Array<string>;
    //日志标题
    title:string;
    //日志内容
    content:string;
    //日志记录时间
    datetime:Date;
    //作者
    author:string;
}